### ISSUE TITLE
genie - not found in I3_PORTS when SYSTEM_PACKAGES=ON (Trac #931)

### ISSUE THREAD

**Issue by icecube-trac** [icecube-trac/tix2#925]
Migrated from https://code.icecube.wisc.edu/ticket/931
```json
{
    "status": "closed", 
    "changetime": "2019-02-13T14:11:45", 
    "description": "", 
    "reporter": "nega", 
    "cc": "", 
    "resolution": "fixed", 
    "_ts": "1550067105393059", 
    "component": "cmake", 
    "summary": "genie - not found in I3_PORTS when SYSTEM_PACKAGES=ON", 
    "priority": "normal", 
    "keywords": "", 
    "time": "2015-04-14T19:54:06", 
    "milestone": "", 
    "owner": "nega", 
    "type": "defect"
}
```

**Comment by icecube-trac** [comment]
Trac update at 20150416T23:42:56: 
* nega changed status from "new" to "closed"
* nega commented:

> In changeset 2131:
> 
> ```text
> #!CommitTicketReference repository="IceTray" revision="2131"
> fall back to I3_PORTS if not found with SYSTEM_PACKAGES
> 
> closes #922, #923, #925, #926
> ```


* nega changed resolution from "" to "fixed"

**Comment by icecube-trac** [comment]
Trac update at 20150416T23:44:11: 
* nega changed status from "closed" to "reopened"
* nega changed resolution from "fixed" to ""
* nega commented:

> erroneously closed.

**Comment by icecube-trac** [comment]
Trac update at 20151209T17:14:17: 
* nega changed status from "reopened" to "accepted"
* nega changed milestone from "" to "IceSim 5.0"

**Comment by icecube-trac** [comment]
Trac update at 20151209T18:32:09: 
* nega commented:

> reopen if this actually affects someone.


* nega changed resolution from "" to "wontfix"
* nega changed status from "accepted" to "closed"

**Comment by icecube-trac** [comment]
Trac update at 20190111T23:52:58: 
* nwhitehorn commented:

> In changeset 170039:
> 
> ```text
> #!CommitTicketReference repository="IceCube" revision="170039"
> fall back to I3_PORTS if not found with SYSTEM_PACKAGES
> 
> closes #922, #923, #925, #926
> ```


* nwhitehorn changed resolution from "wontfix" to "fixed"

**Comment by icecube-trac** [comment]
Trac update at 20190213T14:11:45: 
* olivas commented:

> Milestone IceSim 5.0 deleted


* olivas changed milestone from "IceSim 5.0" to ""
